os/zip/src/os/shaded_org_apache_tools_zip/PermissionUtils.java

```diff
@@ -0,0 +1,153 @@
+/*
```

Comment:
@kiendang do we still need this one vendored file, or can it be removed with all the others?

Reply:
It comes from a different package, `org.apache.tools.ant.utils` instead of `org.apache.tools.zip`, but more importantly it requires modifying manually. There is some code that uses imports not from Java base libraries that needs to be removed. I did put a comment in the file.

Reply:
@kiendang got it!